task/rsync/0.1/rsync.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: rsync
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: rsync, files sync
+    tekton.dev/displayName: "rsync"
```

Comment:
can you please add one more annotation `tekton.dev/categories` and if the image used is a multi-arch image then can you please add `tekton.dev/platforms` annotation?

Reply:
fixed.